django_cloud_deploy/cloudlib/enable_service.py

```diff
@@ -52,7 +55,17 @@ def enable_service_sync(self, project_id: str, service: str):
         service_name = '/'.join(['projects', project_id, 'services', service])
         request = self._service_usage_service.services().enable(
             name=service_name)
-        response = request.execute()
+        try:
+            response = request.execute()
+        except errors.HttpError as e:
+            if e.resp.status == 400:
+                tos = 'terms of service'
+                if tos in str(e):
+                    url = 'https://console.developers.google.com/terms/cloud'
+                    msg = ('Please accept the terms of service in the Google'
+                           'Cloud Console @ {}'.format(url))
+                    raise crash_handling.UserError(msg)
```

Comment:
Maybe "raise crash_handling.UserError(msg) from e". We are still curious about the original exception.

Reply:
What does the message look like if you use `from e` - we want to give the user a simple error message, not a big traceback.

Reply:
I see. @enriquejosepadilla you can ignore my comments here.